addons/web/static/src/webclient/webclient.js

```diff
@@ -78,6 +83,23 @@ export class WebClient extends Component {
             return this.menuService.selectMenu(firstApp);
         }
     }
+
+    /**
+     * @param {MouseEvent} ev
+     */
+    onGlobalClick(ev) {
+        // When a ctrl-click occurs inside an <a href/> element
+        // we let the browser do the default behavior and
+        // we do not want any other listener to execute.
+        if (
+            ev.ctrlKey &&
+            ((ev.target instanceof HTMLAnchorElement && ev.target.href) ||
+                (ev.target instanceof HTMLElement && ev.target.closest("a[href]:not([href=''])")))
+        ) {
+            ev.stopPropagation();
```

Comment:
maybe stopImmediatePropagation also?

Reply:
I don't think it is necessary, this handler is declared on window.
Stopping immediate propagation will only stop other handlers registered on the window object.

But if you prefer I am not truly against it

Reply:
done